db/schema.rb

```diff
@@ -36,17 +36,12 @@
   create_table "allocator", options: "ENGINE=InnoDB DEFAULT CHARSET=utf8", force: :cascade do |t|
     t.string "system_email", null: false
     t.datetime "created"
-    t.integer "doi_quota_allowed", null: false
-    t.integer "doi_quota_used", null: false
-    t.binary "is_active", limit: 1
+    t.binary "is_active", limit: 1, default: "b'1'"
```

Comment:
I am trying to test this locally but it doesn't work. This error is both locally and in travis

> ActiveRecord::StatementInvalid: Mysql2::Error: Invalid default value for 'is_active':

Reply:
This is an issue unrelated to STI and I didn't notice when running locally. Will clean this up.